pybamm/quick_plot.py

```diff
@@ -0,0 +1,227 @@
+import numpy as np
+import pybamm
+import matplotlib.pyplot as plt
+from matplotlib.widgets import Slider
+
+
+class QuickPlot(object):
+    """
+    Generates a quick plot of a subset of key outputs of the model so that the model
+    outputs can be easily assessed. The axis limits can be set using:
+        self.axis["Variable name"] = [x_min, x_max, y_min, y_max]
+    They can be reset to the default values by using self.reset_axis.
+
+    Parameters
+    ----------
+    model: :class: pybamm.BaseModel
+        The model to plot the outputs of.
+    mesh: :class: pybamm.Mesh
+        The mesh on which the model solved
+    solver: :class: pybamm.Solver
+        The numerical solver for the model which contained the solution to the model.
+    """
+
+    def __init__(self, model, param, mesh, solver):
+
+        # geometric parameters
+        self.l_n = param.process_symbol(pybamm.geometric_parameters.l_n).evaluate(0, 0)
+        self.l_s = param.process_symbol(pybamm.geometric_parameters.l_s).evaluate(0, 0)
+        self.l_p = param.process_symbol(pybamm.geometric_parameters.l_p).evaluate(0, 0)
+
+        # spatial and temporal variables
+        self.x_n = np.linspace(0, self.l_n, 40)
+        self.x_p = np.linspace(self.l_n + self.l_s, 1, 40)
+        self.x = np.linspace(0, 1, 100)
+        self.t = solver.t
+
+        self.set_output_variables(solver.y, model, mesh)
+        self.reset_axis()
+
+    def set_output_variables(self, y, model, mesh):
+        self.c_s_n_surf = pybamm.ProcessedVariable(
+            model.variables["Negative particle surface concentration"], self.t, y, mesh
```

Comment:
should the concentration plots be dimensional?

Reply:
I thought about it but decided the numbers are clearer if they aren't, particularly in the particles which will be just be like *20,000 mols/m^3. Whats your thoughts?